.github/workflows/builder_image_tests.yml

```diff
@@ -33,8 +33,10 @@ jobs:
         exclude:
           - quarkus-version: '2.2.3.Final'
             mandrel-builder-image: '20.3-java11'
-          - quarkus-version: '2.4.0.Final'
+          - quarkus-version: '2.5.0.Final'
             mandrel-builder-image: '20.3-java11'
+          - quarkus-version: '2.5.0.Final'
+            mandrel-builder-image: '21.2-java11'
```

Comment:
Do we know `2.5.0` doesn't work with the `21.2` image? I assume we only want tests with `21.3`, but still not sure why we exclude 21.2 now.

Reply:
We know that 2.5.0 no longer supports 21.2, so there is no point in testing it since we won't fix any issues even if they appear (AFAIK it should still work without issues though).

Reply:
OK